src/mscorlib/src/System/IntPtr.cs

```diff
@@ -109,7 +109,16 @@ public unsafe IntPtr(void* value)
     
         [System.Security.SecuritySafeCritical]  // auto-generated
         public unsafe override int GetHashCode() {
-            return unchecked((int)((long)m_value));
+#if FEATURE_CORECLR
+    #if WIN32
+                return unchecked((int)((long)m_value));
+    #else
+                long l = (long)m_value;
+                return (unchecked((int)l) ^ (int)(l >> 32));
+    #endif
+#else
+                return unchecked((int)((long)m_value));
```

Comment:
UIntPtr and IntPtr are fairly inconsistent with each other in regard to `ifdef`ed code. I tried to make the indentation match other places in IntPtr like ToInt32, but then the `ifdef` doesn't line up with that, either.

Is the correct spacing to have the `ifdef`s indented past the method body?